packages/studio/src/ClipPlane.jsx

```diff
@@ -95,7 +95,7 @@ export default function ClipPlane({ clippingPlane, sideColor, children }) {
       node.material = backFaceStencilMat;
     });
 
-    const planeGeom = new THREE.PlaneBufferGeometry();
+    const planeGeom = new THREE.PlaneGeometry();
```

Comment:
Can you explain me what was going on? (I just want to learn - I have no idea why one is better than the other).

Reply:
BTW, thanks a lot for the PR - the first non typo code PR for replicad!

Reply:
Hi @sgenoud, this StackOverflow answer explains it in detail:
https://stackoverflow.com/a/76792805/17515424
It seems they just removed it from Three at some point.

By the way, it is such an awesome library that you created here. Very well done!